io-engine/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -387,6 +418,13 @@ impl<'n> Nexus<'n> {
             .with_block_count(0)
             .with_required_alignment(9)
             .with_data(n)
+            .with_zoned(zone_info.zoned)
```

Comment:
nit: would be tidier to add a with_zone method which takes the zone_info rather than adding all these here 1by1?